bundles/org.openhab.binding.unifi/README.md

```diff
@@ -34,6 +34,7 @@ The following table describes the Bridge configuration parameters:
 | password                 | The password to access the UniFi Controller    | Required | -       |
 | refresh                  | Refresh interval in seconds                    | Optional | 10      |
 
+_Note: port is on Unifi OS (UDM-Pro, ...) normally default  443_
```

Comment:
Updated that for you :)

Reply:
@schinken - thanks! Sorry for being picky, but now when looking at the full formatted README: Since this note is for the port configuration parameter specifically, would it make sense to move it up to the description for that channel? Something like:
```
| port                     | Port of the UniFi Controller. For UniFi OS, the default port is usually 443 | Required | -       |
```

What do you think?

Reply:
done ;)